GOVERNANCE.md

```diff
@@ -0,0 +1,163 @@
+
+## Amundsen Governance Model
+
+Amundsen is a meritocratic, consensus-based community project. Anyone interested in the project can join the community, contribute to the project design, and participate in the decision-making process. This document describes how that participation occurs and how to set about earning merit within the project community.
+
+## Roles And Responsibilities
+
+### Users
+
+Users are community members who need the data discovery features of Amundsen. They are the most important community members, and without them, the project would have no purpose. Anyone can be a user; there are no special requirements.
+
+Amundsen asks its users to participate in the project and community as much as possible. User contributions enable the project team to ensure that they satisfy the needs of those users. Frequent user contributions include (but are not limited to):
+
+- Evangelizing about the project (e.g., a link on a website and word-of-mouth awareness raising)
+- Informing developers of strengths and weaknesses from a new user perspective
+- Providing moral support (a ‘thank you’ goes a long way)
+- Providing financial support (the software is open source, but its developers need to eat)
+
+Users who continue to engage with the project and its community will often become more and more involved. Such users may find themselves becoming contributors, as described in the next section.
+
+### Contributors
+
+Contributors are community members who contribute in concrete ways to the project. Anyone can become a contributor, and contributions can take many forms, as detailed in the [all-contributors project](https://allcontributors.org/docs/en/emoji-key#table). There is no expectation of commitment to the project, no specific skill requirements, and no selection process.
+
+In addition to their actions as users, contributors may also find themselves doing one or more of the following:
+
+- Supporting new users (existing users are often the best people to help new users)
+- Creating, triaging or commenting on Issues
+- Doing code reviews or commenting on technical documents
+- Writing, editing, translating or reviewing the documentation
+- Organizing events or evangelizing the project
+
+Contributors engage with the project through the issue tracker and slack community, or by writing or editing documentation. They submit changes to the project itself via Pull Requests (PRs), which will be considered for inclusion in the project by existing maintainers (see next section). Contributors follow the [Contributing guide](https://lyft.github.io/amundsen/CONTRIBUTING/) when creating PRs.
+
+As contributors gain experience and familiarity with the project, their profile and commitment within the community will increase. At some stage, they may find themselves being nominated for being a maintainer.
+
+### Maintainers
```

Comment:
will there be a place where current list of maintainers is published ?

Reply:
Currently it's [here](https://github.com/lfai/proposing-projects/blob/master/proposals/amundsen.adoc). 
There's also a [PR](https://github.com/lfai/proposing-projects/pull/17) that hasn't been merged yet.

Once it's merged, we can bring that list in the amundsen project too.

Reply:
We will add an owner file to make it explicit.